### ISSUE TITLE
Sample content misleading

### ISSUE THREAD

**Issue by DJBHollis** [dita-ot/dita-ot#2275]
It would appear that the content in the samples folder is misleading.

Take the case that someone has no prior knowledge of DITA. They do a search, and find the toolkit.

They download it, and see what's there.

They find a DITA processor, some instructions and some sample content. They build the sample content.

It works! Brilliant! Isn't DITA easy! All you need is Notepad to hack some DITA files.

They use the content in the samples folder as their guide. Surely this is what DITA content looks like?

Is this far-fetched? It sounds bonkers, but I'm pretty sure it happens.

The toolkit now includes source documents for the user guide.

With a bit of imagination, would it be possible to replace the samples folder with the user guide content?

Instead of a newbie building the samples content, they build the user guide instead, in various different forms.

I'm sure it would be more representative of 'real' DITA content.

**Comment by infotexture** [comment]
We've recognized that the `samples` content was rather outdated, and have begun revising the contents for v2.3 in dita-ot/docs#58 & subsequent commits.

As a first step, we've moved them from the toolkit repository to the docs submodule to make it easier for the docs team to maintain samples, generate new sample content and reference samples via `<coderef>`.

As you mention, the documentation source files are also provided in the `docsrc` folder of the distribution packages, so users can build the documentation via the Ant script provided there. Since the toolkit docs use a few advanced mechanisms to generate topics automatically on build, we haven't yet advocated that approach as an onboarding mechanism for new users.

We plan to describe that process in future versions of the docs (see dita-ot/docs#50) and may consider adding a simplified build example that could be referenced as an additional sample.

Additional feedback & suggestions are welcome under [dita-ot/docs/issues](https://github.com/dita-ot/docs/issues).

**Comment by xephon2** [comment]
:+1: I'd suggest to remove the samples completely and to provide the docs as examples and to explain how the docs are created and built.

**Comment by jelovirt** [comment]
The docs are a realistic example of DITA content, but it's quite a large set of maps and topics to be a sample doc.

**Comment by xephon2** [comment]
No, that is not what I meant. The docs cannot be seen as a single example. May "example" is the wrong word, maybe "design pattern" is better. The sample files could be seen as small snippets to explain common beginner questions like `How do I reference to a topic from a map?`. The docs could be used to explain design patterns like, `How should I organize my files?` or `How should I deploy my docs?`.

The docs contain brilliant design patterns how DITA can be used and how DITA content can be generated and deployed. They are probably the most useful public DITA publishing environment available ant this should be mentioned somewhere.

Maybe the DITA-OT docs are not the correct place for design patterns, but this is what especially newcomers need at most at the beginning. So if the decision should be not to do that, it might be helpful for the users to link them to OASIS whitepapers.

**Comment by infotexture** [comment]
@xephon2 That is the intention of dita-ot/docs#50.

**Comment by xephon2** [comment]
@infotexture Brilliant :smiley:, I haven't seen this issue before.

**Comment by DJBHollis** [comment]
I first downloaded the Toolkit at about OT 1.4. There is, or was, a hierarchical and a sequential build. On the one hand, I was impressed that I could build them. On the other, I didn't understand why there were the two builds. I'm sure the original reason was a simple way to demonstrate topic level reuse, but that was lost on me at the time. I simply thought that this was 'the DITA way'. That DITA implied two ways of viewing content. I probably wondered if this was, somehow, 'expected' from DITA content, although I never have produced two such versions of a document.

Many moons ago, I dabbled with a sophisticated 4GL database tool. At the time, I was wondering how to set about building a database, and creating the UI to interact with it. I thought there had to be 'typical', or 'best practice' way to do it. So I looked at the example content. It just left me confused, because there seemed to be no common denominator, no similarities, nothing to 'latch on to'. Not even a common file naming convention! It took me a while to realise that it probably didn't matter, and one approach was as good as another. But when you're trying something new, particularly something with a steep learning curve, you look for any clue to give you a good start to the first rung.

What is DITA about? It's about creating documentation, in various formats, from a single source. It's customer focussed, and task oriented. It's typically about user manuals, but not exclusively so.

Frankly, is DITA about providing simplistic guidance for menial chores? I don't think so!

Is there, or could there be, a 'quick start guide? A sub-set of the documentation. A 'real' user guide, but a short one. Something with 'proper' sized DITA topics, with a 'sensible' file naming convention. Something for folks to 'latch' on to , and say, "Ah, DITA does that, does it - right! I get it!"
